module_utils/foreman_helper.py

```diff
@@ -209,7 +209,10 @@ def ensure_resource(self, resource, entity_dict, entity, state, check_missing=No
         """ Ensure that a given entity has a certain state """
         changed, changed_entity = False, entity
 
-        if state == 'present':
+        if state == 'present_with_defaults':
```

Comment:
Thanks! I was too lazy to add that when initially doing the Apypie backend, because I didn't *need* it ;)